modules/metastorage/src/testFixtures/java/org/apache/ignite/internal/metastorage/server/SimpleInMemoryKeyValueStorage.java

```diff
@@ -112,14 +120,23 @@ public class SimpleInMemoryKeyValueStorage implements KeyValueStorage {
     private long updCntr;
 
     /**
-     * Last revision of a compact that was set or restored from a snapshot.
+     * Last compaction revision that was set or restored from a snapshot.
      *
      * <p>This field is used by metastorage read methods to determine whether {@link CompactedException} should be thrown.</p>
      *
      * <p>Multi-threaded access is guarded by {@link #mux}.</p>
      */
     private long compactionRevision = -1;
 
+    /**
+     * Last {@link #saveCompactionRevision saved} compaction revision.
+     *
+     * <p>This field is <b>not</b> used by metastorage read methods to determine whether {@link CompactedException} should be thrown.</p>
```

Comment:
I would prefer if you would describe what this field is used for, not what it is *not* used for

Reply:
Try fix it